cmake/toolchain/zephyr/host-tools.cmake

```diff
@@ -7,51 +7,77 @@
 # and currently provided by the Zephyr SDK. Those tools will need to be
 # provided for different OSes and sepearately from the toolchain.
 
-set_ifndef(ZEPHYR_SDK_INSTALL_DIR "$ENV{ZEPHYR_SDK_INSTALL_DIR}")
-set(ZEPHYR_SDK_INSTALL_DIR ${ZEPHYR_SDK_INSTALL_DIR} CACHE PATH "Zephyr SDK install directory")
+set(REQUIRED_SDK_VER 0.11.1)
```

Comment:
Should this be renamed to `MIN_REQUIRED_SDK_VER`?

Reply:
Updated to: `MINIMUM_REQUIRED_SDK_VERSION`